manifest.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Tab Size on GitHub",
-	"version": "1.1.2",
+	"version": "1.1.3",
```

Comment:
Don't bump version in a PR. It's up to the maintainer to decide when to release, what version, and how. I personally use automated script for this.